control/trajectory_follower/design/trajectory_follower-design.md

```diff
@@ -12,9 +12,43 @@ This package provides the library code used by the nodes of the `trajectory_foll
 Mainly, it implements two algorithms:
 
 - Model-Predictive Control (MPC) for the computation of lateral steering commands.
-  - [trajectory_follower-mpc-design](trajectory_follower-mpc-design.md)
+  - [trajectory_follower-mpc-design](mpc_lateral_controller-design.md)
 - PID control for the computation of velocity and acceleration commands.
-  - [trajectory_follower-pid-design](trajectory_follower-pid-design.md)
+  - [trajectory_follower-pid-design](pid_longitudinal_controller-design.md)
+
+## Design
+
+![Controller](../../trajectory_follower_nodes/design/media/Controller.drawio.svg)
```

Comment:
Use UML default way for this diagram.

Reply:
updated in https://github.com/autowarefoundation/autoware.universe/pull/1591/commits/5cdadaff82f4ef1f8f24a692c2b4a9a28a3fe716